esda/moran.py

```diff
@@ -217,7 +218,7 @@ def __moments(self):
         self.VI_norm = v_num / v_den - (1.0 / (n - 1)) ** 2
         self.seI_norm = self.VI_norm ** (1 / 2.0)
 
-        # variance under randomization
+        # variance under total randomization
```

Comment:
This change can get dropped; it's fine to just say "randomization" for the global stat cause there's no alternative randomization strategy.